src/app-nordigen/banks/integration-bank.js

```diff
@@ -33,6 +33,10 @@ export default {
         top10SortedTransactions: JSON.stringify(sortedTransactions.slice(0, 10))
       }
     );
-    return 0;
+
+    const currentBalance = balances[0];
```

Comment:
Let's use here the priority of balance types. WDYT? IMO we should base if it's possible on the booked transactions, not on which are expected.
https://nordigen.com/en/docs/account-information/output/balance/

Reply:
Updated the PR.

Is this what you meant?

Reply:
Yes, I'm not sure if it's the correct order but I'm not an expert in the account types :D I believe we will have a possibility to tweak it if needed.
IMO the most important is to have at the end of this list balance types based on the not booked transactions because they can just disappear in the meantime. On another side, we use this function only during adding a new account.

Reply:
I just double checked with my Latvian banks and it turns out we do need `expected` balanceType in the list for things to work. Without it my balance is calculated totally wrong.

https://github.com/actualbudget/actual-server/pull/144/commits/594df22a9d2a6f1d7e14fe41e7e26c744c6b2834